fp.js

```diff
@@ -6,10 +6,13 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
-  peoplesAges.push(person.age)
-})
+  var peoplesAges = people.map(function(obj) {
+    var isolated = {};
+    isolated[obj.age] = obj.value
+    return obj.age;
+  });
```

Comment:
This works, although all you need to do with `map` is return the value you want to see in the new array. So something like this...

```js
const ages = people.map(person => {
  // We just need each person's age in our new array, so let's just return that.
  return person.age;
})
```